.env.example

```
@@ -0,0 +1,2 @@
+MAPBOX_ACCESS_TOKEN="xyz"
+MAP_SERVICES_API_URL="https://api.mapbox.com"
```

Map services should point to a [Valhalla](https://github.com/valhalla/valhalla) instance. We could have a public instance as a default to highlight this:

```suggestion
MAP_SERVICES_API_URL="https://valhalla1.openstreetmap.de"
```